cuda2gcn/src/math.cl

```diff
@@ -0,0 +1,279 @@
+/*===--------------------------------------------------------------------------
+ *                   ROCm Device Libraries
+ *
+ * This file is distributed under the University of Illinois Open Source
+ * License. See LICENSE.TXT for details.
+ *===------------------------------------------------------------------------*/
+
+#include "ocml.h"
+
+#define ATTR __attribute__((always_inline const))
```

Comment:
The OCML functions themselves are not all const.  I'd drop the const here for now.